secure-custom-fields.php

```diff
@@ -741,6 +752,19 @@ public function acf_plugin_activated() {
 				}
 			}
 		}
+
+		/**
+		 * Initializes the ACF WooCommerce HPOS integration.
+		 *
+		 * @since 6.5
+		 *
+		 * @return void
+		 */
+		public function init_hpos_integration() {
+			acf_include( 'includes/forms/form-wc-order.php' );
```

Comment:
Should we check if the `custom_order_tables` feature is enabled in WooCommerce before including these?

Reply:
The functions already have a backup method if the feature is not enabled.